tools/sense_studio/utils.py

```diff
@@ -12,13 +12,15 @@
 from tools import directories
 from tools.sense_studio.project_utils import get_project_setting
 
-SUPPORTED_MODEL_CONFIGURATIONS = [
+ALL_SUPPORTED_MODEL_CONFIGURATIONS = [
     ModelConfig('StridedInflatedEfficientNet', 'pro', []),
     ModelConfig('StridedInflatedMobileNetV2', 'pro', []),
     ModelConfig('StridedInflatedEfficientNet', 'lite', []),
     ModelConfig('StridedInflatedMobileNetV2', 'lite', []),
 ]
 
+SUPPORTED_MODEL_CONFIGURATIONS = [i for i in ALL_SUPPORTED_MODEL_CONFIGURATIONS if i.model_exist() is True]
```

Comment:
This is only evaluated once when the python stack is started. I would prefer to check that live whenever the list of models is requested, because a user might have downloaded the models in the meantime.